ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-master.yaml

```diff
@@ -61,7 +65,7 @@ tests:
     cluster_profile: aws
 - as: hco-e2e-upgrade-prev-aws
   commands: |
-    RELEASE_DELTA=2 make upgrade-test
+    RELEASE_DELTA=2 PREV=true make upgrade-test
```

Comment:
there is no `PREV` variable in `upgrade-test.sh` executed by `make upgrade-test`.
maybe you meant to use `upgrade-create-new-version.sh`?

Reply:
No. `upgrade-test.sh` calls `upgrade-openshiftci-config`. See here: https://github.com/kubevirt/hyperconverged-cluster-operator/blob/a97db1a7ccba28e48cc13e26efb9e7938542e418/hack/upgrade-test.sh#L78 and here: https://github.com/kubevirt/hyperconverged-cluster-operator/blob/a97db1a7ccba28e48cc13e26efb9e7938542e418/hack/upgrade-openshiftci-config#L7-L11